sparkle/src/components/Searchbar.tsx

```diff
@@ -24,20 +27,30 @@ export function Searchbar({
   disabled?: boolean;
   className?: string;
 }) {
+  const inputRef = useRef<HTMLInputElement>(null);
```

Comment:
this inputref is not assigned so not useful; can be safely removed along with the two lines below referencing it